### ISSUE TITLE
Contact avatar updates not immediately

### ISSUE THREAD

**Issue by Simon-Laux** [deltachat/deltachat-android#421]
In our test we restarted the app after adding the contact in the phones contact app, but the avatar updated only after we went to the create new chat menu. 

Also it might be a good idea to be able to set contact profile pictures right out of the app even if it creates the contact on the phone.

**Comment by csb0730** [comment]
> Also it might be a good idea to be able to set contact profile pictures right out of the app even if it creates the contact on the phone.

Good proposal :-)

**Comment by r10s** [comment]
trying to sort things in this issue a bit:

> In our test we restarted the app after adding the contact in the phones contact app, but the avatar updated only after we went to the create new chat menu.

instant-update is out of scope for now. the system address book is just updated when the contacts-list is read, changing this would be quite some effort. also, in the mid-term, the general idea is more to let the sender specify the profile image that is then send to contacted recipients, see https://support.delta.chat/t/set-local-profile-pictures/444 - which has the advantage that this is available on all systems equally.

> Also it might be a good idea to be able to set contact profile pictures right out of the app even if it creates the contact on the phone.

same here - the idea is more to let the sender decide how to appear on the others phone.

> Also if you change the contact name inside Delta Chat while chatting with that contact, the contact name isn't updated immediately on the top bar, you have to go back to the chats list first.

this is fixed meanwhile.
